src/applications/personalization/profile/tests/components/ProfileWrapper.unit.spec.js

```diff
@@ -34,21 +34,21 @@ describe('ProfileWrapper', () => {
       .not.to.be.null;
   });
 
-  it('should render NameTag when the full name of the user was fetched)', () => {
-    const initialState = {
-      vaProfile: {
-        hero: {
-          userFullName: {
-            first: 'Test',
-            last: 'Test',
-          },
-        },
-      },
-    };
-    const { getByTestId } = render(ui, { initialState });
-    const NameTag = getByTestId('name-tag');
-    expect(NameTag.textContent.match(/Test Test/i)).not.to.be.null;
-  });
+  // it('should render NameTag when the full name of the user was fetched)', () => {
```

Comment:
nit: could we do `it.skip(` instead of commenting out all the lines?

Reply:
Ah, good point. Will remember for next time - I'm reluctant to restart the build so it can get merged fast. Looking to get this fixed asap.

Reply:
err, but if this passes CI just merge it!